srtcore/buffer_tools.h

```diff
@@ -188,14 +187,53 @@ class CSndRateEstimator
         bool empty() const { return m_iPktsCount == 0; }
     };
 
-    int incSampleIdx(int val, int inc = 1) const;
+private:
+    static const int NUM_PERIODS        = 10;
+    static const int SAMPLE_DURATION_MS = 100; // 100 ms
```

Comment:
These can be made just const (drop `static`) and initialized accordingly in `CSndRateEstimator` and `CMobileRateEstimator`.

Reply:
Ok, I just take as it was in the mother class. I remove it.

Reply:
If it's not static, I can't initialize the table directly in the class.

Reply:
Right. In that case you would need a dymanic array that can be initialized from a constructor. Usually in C++ the best candidate is `std::vector`. In this case we have a more strict class `srt::FixedArray` that fits better as it does not allow resizing.

Reply:
Ok, I'll see this, thanks.